frontend/heritage-trail-game/heritage-trail-game.css

```
@@ -1,454 +1,452 @@
+  background: var(--tp-panel); backdrop-filter: blur(10px);
+  padding: 18px 20px; margin-bottom: 22px; box-shadow: var(--tp-shadow);
+}
+.control-group { display: grid; gap: 8px; }
+.control-label { font-size: 0.7rem; font-weight: 900; text-transform: uppercase; letter-spacing: 0.1em; color: var(--tp-muted); }
+.mode-hint { text-transform: none; letter-spacing: 0; font-weight: 600; font-style: italic; }
+
+.seg { display: inline-flex; gap: 4px; border: 1px solid var(--tp-line); border-radius: 999px; padding: 4px; background: var(--tp-panel-2); }
+.seg button {
+  display: grid; justify-items: center; line-height: 1.15;
+  border: 0; background: transparent; color: var(--tp-muted);
+  border-radius: 999px; padding: 8px 16px; font: inherit; font-weight: 800; cursor: pointer;
```

Stylelint (17.14.0)

[error] Overridden property "line-height" by shorthand "font" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)